### PR TITLE
Feature/SDK-484 update dev site mp point 

### PR DESCRIPTION
**Pull request by etienne-pradere**
## Description
Se realiza la actualización de la documentación del menú Pagos presenciales > Mercado Pago Point.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [X] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [X] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [X] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [X] Requires translations.

### CODE REVIEW

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 1]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
+> 
+> Dispositivos soportados con esta integración:
+> 
+> - [Point Bluetooth](https://www.mercadopago.com.ar/point-bluetooth?ref=devsite)
+------------
 
-Existen dos grandes mundos a la hora de integrar Point:
+> WARNING
+>
+> Para poder cobrar de manera integrada con nuestro dispositivo Point es necesario descargar la aplicación de Mercado Pago disponible en los marketplaces de iOS y Android.
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:21]:
```suggestion
> Para poder cobrar de manera integrada con nuestro dispositivo Point es necesario descargar la aplicación de Mercado Pago disponible en las tiendas de aplicaciones de iOS y Android.
```

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 2]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
+ 
+> [<img src="/mobile/GooglePlayBadge.es.png" alt="Android Play Store" width="200"/>](https://play.google.com/store/apps/details?id=com.mercadopago.wallet&hl=es_419)
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:23]:
```suggestion
> 
```

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 3]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
+> [<img src="/mobile/AppStoreBadge.es.svg" alt="iOS App Store" width="158" style="margin:0.8em"/>](https://itunes.apple.com/ar/app/mercado-pago/id925436649?mt=8)
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:24]:
```suggestion
> 
```

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 4]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
+
+**Características:**
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:26]:
```suggestion
## Características:
```

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 5]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
 
-1) Cuando la aplicación se puede utilizar desde el mismo dispositivo (celular o tablet) donde esta instalada la aplicación de Mercado Pago. Esto se puede lograr con una integración de deep linking o intent-based mediante
+* Integrando tu aplicación la modalidad de cobros vía Point. Tu aplicación genera una orden de pago con el monto y una descripción y llama a la aplicación de Mercado Pago para que el vendedor directamente deslice la tarjeta del pagador y cobre. Una vez procesado el pago, se retorna a tu aplicación.
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:28]:
```suggestion
* **Integrando tu aplicación la modalidad de cobros vía Point**: Tu aplicación genera una orden de pago con el monto y una descripción y llama a la aplicación de Mercado Pago para que el vendedor directamente deslice la tarjeta del pagador y cobre. Una vez procesado el pago, se retorna a tu aplicación.
```

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 6]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
+
+* Integrando a tu sistema de gestión la modalidad de cobros vía Point. Desde tu sistema se genera una orden de cobro con el monto, una descripción y el dispositivo que quieras que reciba dicha orden. Automáticamente se levantará la aplicación de Mercado Pago para que el vendedor directamente deslice la tarjeta del pagador y cobre. Una vez procesado el pago, se cerrará la orden.
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:30]:
```suggestion
* **Integrando a tu sistema de gestión la modalidad de cobros vía Point**: desde tu sistema se genera una orden de cobro con el monto, una descripción y el dispositivo que quieras que reciba dicha orden. Automáticamente se levantará la aplicación de Mercado Pago para que el vendedor directamente deslice la tarjeta del pagador y cobre. Una vez procesado el pago, se cerrará la orden.
```

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 7]
```diff
@@ -6,16 +6,34 @@ sites_supported:
...
+
+Existen dos grandes mundos a la hora de integrar Point:
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:32]:
```suggestion
## Dos maneras de integrar Point
```

Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:32]:
```suggestion
**Existen dos grandes mundos a la hora de integrar Point**:
```

**Comment by abderossi** [comment]
Cerramos este PR momentáneamente. Lo esperamos nuevamente cuando lo trabajemos en conjunto en Q4 para publicarlo a mar abierto.

**Comment by stevenossameli** [comment]
Hola @abderossi , quedamos pendientes por si surgen comentarios respecto a este DRAFT.

**Pull request closed without merging**